packages/augur-sdk/src/connector/seo-connector.ts

```diff
@@ -1,11 +1,59 @@
 import * as Sync from '../state/Sync';
-import { Subscriptions } from '../subscriptions';
 import { SingleThreadConnector } from './single-thread-connector';
+import { SubscriptionEventName } from '../constants';
+import { Callback, SubscriptionType } from '../events';
 
 export class SEOConnector extends SingleThreadConnector {
   connect = async (ethNodeUrl: string, account?: string): Promise<any> => {
     this.api = Sync.start(ethNodeUrl, account, true);
-    this.events = (await this.api).augur.getAugurEventEmitter();
+    this.events = this.api.then(api => api.augur.getAugurEventEmitter());
```

Comment:
The use of `fetch` instead of `await` ensures that the connector will have what it needs before moving forward.

Using `await` will make lots of things to go `undefined`.

Reply:
Omfg. That was really tricky.